seip2020_practical_assignments/math/src/main/java/math/ArrayOperations.java

```diff
@@ -0,0 +1,31 @@
+package math;
+
+import java.util.ArrayList;
+
+import io.FileIO;
+
+
```

Comment:
Missing @author and class description javadoc. Apply that to other classes too. Not required for test classes.